cmd/nvidia-container-runtime-hook/container_config_test.go

```diff
@@ -477,9 +477,19 @@ func TestGetNvidiaConfig(t *testing.T) {
 	}
 	for _, tc := range tests {
 		t.Run(tc.description, func(t *testing.T) {
-			image, _ := image.New(
+			opts := []image.Option{
```

Comment:
Don't the results of this test ONLY depend on the CUDA image implementation? If this is the case, they should be moved to the `config/image` package.